test_communication/test/test_secure_publisher_subscriber.py.in

```diff
@@ -0,0 +1,60 @@
+import os
```

Comment:
done in https://github.com/ros2/system_tests/pull/204/commits/baa6d1bd80b8f9d805b943c93dbead00f75c943a